tests/test_client.py

```diff
@@ -0,0 +1,244 @@
+# import backoff
+import os
+import random
+import requests
+import string
+
+# from requests.exceptions import HTTPError
+# from requests.auth import HTTPBasicAuth
+
+# TODO try below syncronous interaction or go all http?
+# from facebookads.objects import Ad, AdAccount, AdSet, Campaign
+
+from tap_tester.logger import LOGGER
+
+
+class TestClient():
+    # def __init__(self, config):  # TODO move to dynamic config model?
+    def __init__(self):
+        # pass in config above and get() from it or hard code?
+        self.base_url  = 'https://graph.facebook.com/'
+        self.api_version = 'v18.0'
+        self.account_id = os.getenv('TAP_FACEBOOK_ACCOUNT_ID')
+        self.access_token  = os.getenv('TAP_FACEBOOK_ACCESS_TOKEN')
+        self.account_url = self.base_url + self.api_version +'/act_{}'.format(self.account_id)
```

Comment:
We should try to move away from `.format()` calls with new code. Every modern python version supports F-strings and any version that doesn't is deprecated

```suggestion
        self.account_url = f"{self.base_url}/{self.api_version}/act_{self.account_id}"
```
This also lets us make the decision that no piece of the url should have a trailing or leading slash and we can let whatever code builds a full URL add the slashes in for us.

Reply:
Also, if we never separate `'https://graph.facebook.com/'` from `'v18.0'`, then I think it's safe to say that the `base_url` is `'https://graph.facebook.com/v18.0'`. 
1. It makes the "build a full URL" lines shorter
2. It makes it harder to deviate from the one API version this client uses

Reply:
Not sure if we should tie to one api version so updated accordingly